addons/website_gengo/controllers/main.py

```diff
@@ -37,3 +37,30 @@ def set_gengo_config(self,config):
     def post_gengo_jobs(self):
         request.registry['base.gengo.translations']._sync_request(request.cr, request.uid, limit=GENGO_DEFAULT_LIMIT, context=request.context)
         return True
+
+    @http.route('/website/set_translations', type='json', auth='user', website=True)
```

Comment:
Either this controller should be moved to website or if it is used only for website_gengo then route should start with website_gengo